.github/workflows/drivers_ci.yml

```diff
@@ -275,16 +275,38 @@ jobs:
           retention-days: 1
 
   build-scap-open-w-extern-bpf-skeleton:
+    env:
+      ACTIONS_ALLOW_USE_UNSECURE_NODE_VERSION: true
     needs: [paths-filter,build-modern-bpf-skeleton] 
     # See https://github.com/actions/runner/issues/409#issuecomment-1158849936
     runs-on: ${{ (inputs.arch == 'aarch64' && 'actuated-arm64-8cpu-16gb') || 'ubuntu-latest' }}
     if: needs.paths-filter.outputs.driver == 'true' || needs.paths-filter.outputs.libscap == 'true' || needs.paths-filter.outputs.libpman == 'true'
     container: centos:7
     steps:
       # Always install deps before invoking checkout action, to properly perform a full clone.
-      - name: Install build dependencies
+      - name: Fix mirrors to use vault.centos.org
```

Comment:
IMO we can remove centos7 here since in libs the CI doesn't release any official package, we don't need to support very old GLIBC versions, something like Ubuntu 18:04 or similar should be fine :thinking: WDYT?

Reply:
Yep but this was created to mimic Falco CI to catch issues before they hit Falco repo :)